app/Domains/Contact/ManageContact/Web/Controllers/ContactLabelController.php

```diff
@@ -18,7 +18,7 @@ public function index(Request $request, string $vaultId, int $labelId)
     {
         $vault = Vault::findOrFail($vaultId);
 
-        $contacts = Label::where('id', $labelId)->first()
+        $contacts = Label::find($labelId)
```

Comment:
Why `find` and not `first`?

Reply:
`find(xx)` is actually the same as `where('id', xxx)->first()`